featurebyte/session/base.py

```diff
@@ -1265,9 +1284,7 @@ async def update_metadata_schema_version(self, new_version: int) -> None:
         new_version : int
             New version to update the working schema to
         """
-        update_version_query = (
-            f"""UPDATE METADATA_SCHEMA SET WORKING_SCHEMA_VERSION = {new_version}"""
-        )
+        update_version_query = f"""UPDATE {self.session.metadata_schema} SET WORKING_SCHEMA_VERSION = {new_version} WHERE 1=1"""
```

Comment:
What is the reason to add the `1=1` condition?

Reply:
BigQuery requires a condition for UPDATE statements